packages/snap-preact-demo/tests/cypress/support/commands.js

```diff
@@ -114,3 +114,12 @@ Cypress.Commands.add('waitForIdle', (options) => {
 		});
 	});
 });
+
+Cypress.Commands.add('waitForRecsReady', () => {
+	return cy.document().then((doc) => {
+		return new Cypress.Promise((resolve) => {
+			let timeout = setTimeout(resolve, 5000);
```

Comment:
Is this timeout required? I guess you are just giving a max time of 5000 to render the images before auto-resolving?

Reply:
right its just a max time back up incase that event never fires.